src/Routes.js

```diff
@@ -37,117 +33,66 @@ const TopicAdmin = lazy(() =>
     /* webpackChunkName: "TopicAdmin" */ './PresentationalComponents/TopicsAdminTable/TopicsAdminTable'
   )
 );
-const AdminProtectedRoute = lazy(() =>
```

Comment:
This import is also needed as this component is rendered on line 89. Also, the component might need to be changed to TopicAdmin as on line 89.

Reply:
It is present. Check the line 153